bridge/README.md

```diff
@@ -131,6 +131,10 @@ pip install -e .
 
 In addition, you can use any standard `boto3` or MlFlow environment variables.
 
+If you do use any new variables not listed above and
```

Comment:
This is hard to parse, I think it needs a comma. I'm also not sure what it is asking. Are we asking users to help us maintain the README?

Reply:
Eh, deleting.